internal/extsvc/bitbucketserver/events.go

```diff
@@ -54,16 +54,28 @@ type Webhook struct {
 	Secret   string   `json:"secret"`
 }
 
+const webhookURL = "rest/sourcegraph-admin/1.0/webhook"
```

Comment:
```suggestion
const webhookEndpoint = "rest/sourcegraph-admin/1.0/webhook"
```

... or `webhookPath`?